src/CashuMint.ts

```diff
@@ -48,7 +48,28 @@ class CashuMint {
 		customRequest?: typeof request
 	): Promise<GetInfoResponse> {
 		const requestInstance = customRequest || request;
-		return requestInstance<GetInfoResponse>({ endpoint: joinUrls(mintUrl, '/v1/info') });
+		const data = await requestInstance<GetInfoResponse>({
+			endpoint: joinUrls(mintUrl, '/v1/info')
+		});
+		// BEGIN DEPRECATED
+		// Monkey patch old contact field ["email", "me@mail.com"] Array<[string, string]>; to new contact field [{method: "email", info: "me@mail.com"}] Array<MintContactInfo>
+		// This is to maintain backwards compatibility with older versions of the mint
+		if (Array.isArray(data?.contact) && data?.contact.length > 0) {
+			data.contact = data.contact.map((contact: any) => {
+				if (
```

Comment:
I have the feeling that this check id a bit to verbose. We should assume that mint info responses implement either the old way or the new way.

Reply:
I don't understand the issue.